test/apicoverage/image/kodata/ignoredfields.yaml

```diff
@@ -18,9 +18,43 @@
   fields:
     - Name
     - VolumeDevices
+    - Stdin
+    - Lifecycle
+    - TTY
+    - StdinOnce
+    - TerminationMessagePolicy
```

Comment:
Lets go ahead and put TerminationMessagePolicy and TerminationMessagePath back in as it is an allowed field.

Reply:
Done